<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceListPrinterGUI</class>
 <widget class="QDialog" name="PriceListPrinterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stampa Elenco Prezzi su file ODT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="priceDataSetGroupBox">
       <property name="title">
        <string>Preventivo</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="priceDataSetSpinBoxLabel">
          <property name="text">
           <string>N. Preventivo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="priceDataSetSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="paperGroupBox">
       <property name="title">
        <string>Supporto</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QGroupBox" name="paperOrientationGroupBox">
          <property name="title">
           <string>Orientamento</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QRadioButton" name="paperHorizzontalRadioButton">
             <property name="text">
              <string>Orizzontale</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="paperVerticalRadioButton">
             <property name="text">
              <string>Verticale</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="paperDImensionsGroupBox">
          <property name="title">
           <string>Dimensioni</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QComboBox" name="paperDimensionsComboBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>167</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="2" colspan="4">
    <widget class="QGroupBox" name="priceFieldsGroupBox">
     <property name="title">
      <string>Campi Prezzo</string>
     </property>
     <layout class="QGridLayout" name="priceFieldsGroupBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <layout class="QGridLayout" name="priceFieldPushButtonLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="insertPriceFieldPushButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="removePriceFieldPushButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="piceFieldPushButtonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="priceFieldComboBoxLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="printPriceAPGroupBox">
       <property name="title">
        <string>Documenti da stampare</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="printPriceListCheckBox">
          <property name="text">
           <string>Elenco Prezzi</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="printAPCheckBox">
          <property name="text">
           <string>Analisi Prezzi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ApgroupPrAmCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Raggruppa Prezzi e Importi delle Analisi</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Descrizione voci di prezzo</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QRadioButton" name="printShortDescRadioButton">
          <property name="text">
           <string>Descrizione breve</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="printLongDescRadioButton">
          <property name="text">
           <string>Descrizione estesa</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="printShortLongDescRadioButton">
          <property name="text">
           <string>Descrizione breve ed estesa</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="printShortLongDescOptRadioButton">
          <property name="text">
           <string>Descrizione breve ed estesa (ottimizzato)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PriceListPrinterGUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PriceListPrinterGUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printAPCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ApgroupPrAmCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
